batch/gcp-create-worker-image.sh

```diff
@@ -36,7 +36,7 @@ create_build_image_instance() {
     gcloud -q compute instances create ${BUILDER} \
         --project ${PROJECT}  \
         --zone=${ZONE} \
-        --machine-type=n1-standard-1 \
+        --machine-type=n1-standard-4 \
```

Comment:
is this an intentional change? cc: @daniel-goldstein

Reply:
Not that I'm aware of.

Reply:
Oh yeah I had changed this when my startup script was taking a long time to build to see if it helped. The long build ended up being because of an error so I'll revert this